app/routes/goal_routes.py

```diff
@@ -1 +1,119 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response
+from flask import request, abort, make_response
+from ..models.goal import Goal
+from ..models.task import Task
+from .route_utilities import validate_model, create_model, get_models_with_filters
+from ..db import db
+from sqlalchemy import asc, desc
+from flask import jsonify
+
+
+bp = Blueprint("goals_bp", __name__, url_prefix = "/goals")
+
+@bp.post("")
+def create_new_goal():
+    request_body = request.get_json()
+    if (
+        not request_body
+        or "title" not in request_body
+    ):
+        abort(make_response({"details": "Invalid data"}, 400))
+    new_goal = Goal.from_dict(request_body)
+    db.session.add(new_goal)
+    db.session.commit()
+
+    return {
+            "goal":{
+            "id" : new_goal.id,
+            "title": new_goal.title
+            }
+        }, 201
```

Comment:
You wrote `create_model` in `routes_utilities.py` that could replace almost all of the code in this route. Invoking the helper function here would make this route more concise and also cut down on repeated code in the codebase. 

```suggestion
    create_model(Goal, request_body)
```